doc/userguide/rules/sip-keywords.rst

```diff
@@ -0,0 +1,179 @@
+SIP Keywords
+============
+
+The SIP keywords are impletemented as sticky buffers can be used to match on fields in SIP messages.
```

Comment:
The SIP keywords are implemented as sticky buffers and can be used to match on fields in SIP messages.